guides/source/action_text_overview.md

```diff
@@ -112,6 +112,110 @@ class MessagesController < ApplicationController
 end
 ```
 
+## Rendering Rich Text content
+
+ActionText will sanitize and render rich content on your behalf.
+
+By default, the Action Text editor and content is styled by the Trix defaults.
+
+If you want to change these defaults, you'll want to remove the
+`app/assets/stylesheets/actiontext.scss` linker and base your stylings on the
+[contents of that file](https://raw.githubusercontent.com/basecamp/trix/master/dist/trix.css).
+
+By default, ActionText will render rich text content into an element that
+declares the `.trix-content` class:
+
+```html+erb
+<%# app/views/layouts/action_text/contents/_content.html.erb %>
+<div class="trix-content">
+  <%= yield %>
+</div>
+```
+
+If you'd like to change the rich text's surrounding HTML with your own layout,
+declare your own `app/views/layouts/action_text/contents/_content.html.erb`
+template and call `yield` in place of the content.
+
+You can also style the HTML used for embedded images and other attachments
+(known as blobs). On installation, Action Text will copy over a partial to
+`app/views/active_storage/blobs/_blob.html.erb`, which you can specialize.
+
+### Rendering attachments
```

Comment:
This is awesome part. I always wanted to document this as currently their is no way anyone would know how to process and list attachments. Thank you @seanpdoyle